source/_includes/code_examples/tab.liquid

```diff
@@ -1,3 +1,3 @@
-<li class="ui-tabs-tab{% if syntax == 'css' %} css-tab{% endif %}{% if enabled %} ui-tabs-active{% endif %}">
+<li class="ui-tabs-tab{% if syntax == 'css' %} css-tab{% endif %}{% if enabled %} ui-tabs-active{% endif %}" data-syntax="{{ syntax }}">
```

Comment:
What is this data attribute for?

Reply:
Outdated -- removed in https://github.com/sass/sass-site/pull/1167/commits/4612ff139916ba78965ed2732f76c67a721b261f